connector-builder-server/src/main/openapi/openapi.yaml

```diff
@@ -61,53 +63,71 @@ components:
     StreamRead:
```

Comment:
@girarda @brianjlai @sherifnada 
I've made a few changes to the StreamRead object here, if you could take a look. The main thing is that I've split out `airbyteMessages` into separate fields for `logs`, `records`, and `state`. 

The reasoning behind this is mainly coming from not being able to pull in the AirbyteProtocol struct into this openAPI spec, so we will need the connector-builder-server to filter out the various types of AirbyteMessages into separate fields like this so that the frontend knows what to do with them.

I pulled `logs` up to the top level of this response, since I feel like users will just want to view all of the logs for the entire read operation (and that also matches the "logs tray" design in the prototype), rather than seeing them split up by page. But let me know if you disagree.

I also pulled `state` up to the slice-level of this response, since we expect that there should be one state message for each slice.

Reply:
I agree with moving the state to the slice-level.

No strong opinion on whether the the logs should be at the page-level or global. I think global is fine

Reply:
also seems fine to me. We sometimes have to deal w/ consecutive state messages (or the case where slices have no records) so maybe this approach works better anyway so the underlying python server can obfuscate the complexity of formatting what some of these outputs might look like